### PR TITLE
Err out if we want events that are not there

### PR DESCRIPTION
**Pull request by simonjohansson**
Error out instead of keep going with default event if we specify an event that is not there.

### CODE REVIEW

File: events/events.go [hunk 1]
```diff
@@ -58,20 +58,22 @@ func routeEvent(msg *events.Envelope) {
...
-	for _, event := range strings.Split(wantedEvents, ",") {
-		if isAuthorizedEvent(event) {
-			selectedEvents[event] = true
-			log.LogStd(fmt.Sprintf("Event Type [%s] is included in the fireshose!", event), false)
-		} else {
-			log.LogError(fmt.Sprintf("Rejected Event Name [%s] - See wanted/selected events arg. Check Your Command Line Arguments!", event), "")
-		}
-	}
-	// If any event is not authorize we fallback to the default one
-	if len(selectedEvents) < 1 {
```
Comment (shinji62) [inline events/events.go]: This one is important let s say we want an event but is not in the list ?

Comment (simonjohansson) [inline events/events.go]: @shinji62 care to elaboreate, Im not following im afraid.

Comment (shinji62) [inline events/events.go]: No pb dude, let' say I put --events=badEventName what's happen ?
selectedEvents is empty ... so we should fallback to LogMessage.

Comment (simonjohansson) [inline events/events.go]: Ah, thanks.

With this PR if you enter a bad event name, in any place in --events firehose-to-syslog will exit -1 and print a error message "Rejected Event Name [%s] ... ".

I think this is a better approach than falling back to a "default" event, so there is no ambiguity as to why you get an event you did not ask for and why you dont get what you actually wanted.

Also, just noticed, the error message should be better, currently

Comment (shinji62) [inline events/events.go]: Ok got it make sens.

**Comment by shinji62** [comment]
LGTM

**Pull request merged**